scylla-cql/src/deserialize/value.rs

```diff
@@ -675,12 +675,14 @@ where
 // collections
 
 make_error_replace_rust_name!(
+    pub(crate),
     typck_error_replace_rust_name,
     TypeCheckError,
     BuiltinTypeCheckError
 );
 
 make_error_replace_rust_name!(
+    pub,
     deser_error_replace_rust_name,
     DeserializationError,
```

Comment:
:question: Why is once of these functions used by external code and the other is not? They seem to be very similar, so I see no immediate reason.
If they are so similar, I reason that they should have the same visibility. Same goes for `row.rs` replace functions.

Reply:
`typck_error_replace_rust_name` is only used in `DeserializeValue` impls written in the `value.rs` file.
`deser_error_replace_rust_name` is also used in those impls, but also in `scylla-macros` in generated deserialize method impls.
Why? I'm not sure, and its not easy to read the macro code - since you've written it I invite you to check that.